pkg/machine/hyperv/machine.go

```diff
@@ -513,7 +516,15 @@ func (m *HyperVMachine) Stop(name string, opts machine.StopOptions) error {
 	if vm.State() != hypervctl.Enabled {
 		return hypervctl.ErrMachineStateInvalid
 	}
+
+	defer func() {
+		if err := machine.CleanupGVProxy(m.GvProxyPid); err != nil {
+			logrus.Error(err)
```

Comment:
Why is this defer? just call vm.Stop() followed by this.

The problem with defer is that you now only log the error which causes us to still exit 0 which is bad for CI testing and use in scripts.